incubator-tools/cmek_docai_processor/cmek_docai_processor.ipynb

```diff
@@ -0,0 +1,909 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    source_version = "projects/{project_number}/locations/{location}/processors/{processor_id}/processorVersions/{processor_version}"
</pre><p>This (and all similar strings) should be constructed through clients instead of hardcoding the string format.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/749/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>